cmd/logger/logger.go

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-package cmd
+package logger
 
 import (
+	"context"
```

Comment:
the license of this file should reflect 2018

Reply:
done